install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1668,6 +1668,13 @@ prometheus:
   # ref: https://docs.cilium.io/en/stable/operations/metrics/#exported-metrics
   metrics: ~
 
+dashboards:
```

Comment:
Could you leave some doc comments here to document the meaning of the flags?

Reply:
I can't seem to find the grafana documentation that describes the `grafana_dashboard` import configmap....

Reply:
[Sidecar for dashboards](https://github.com/grafana/helm-charts/tree/main/charts/grafana#sidecar-for-dashboards) section in Grafana Helm chart docs documents that

Reply:
Thanks for the link!

In theory I've got doc updated with comments and links.